packages/common/src/types/api/sort.ts

```diff
@@ -0,0 +1,2 @@
+export type ApiSortDirection = 'asc' | 'desc';
+export type ApiSort = { sort: string; order?: ApiSortDirection };
```

Comment:
we should probably add the possible strings `sort` can be for now? like the columns of the catalog or `'name'` for now? wdyt?